common/remote/sockfile.cpp

```diff
@@ -363,13 +363,95 @@ const char *RFCStrings[] =
     RFCText(RFCsetfileperms),
     RFCText(RFCunknown),
 };
-static const char *getRFCText(RemoteFileCommandType cmd)
+static const char *getRFCText(unsigned cmd)
```

Comment:
There's no longer a reason for this change is there? (RemoteFileCommandType to unsigned)